src/mantine-core/src/components/Select/Select.tsx

```diff
@@ -287,6 +287,9 @@ export const Select = forwardRef<HTMLInputElement, SelectProps>(
       );
     }, [inputValue]);
 
+    const getSelectedItemIndex = () =>
```

Comment:
Is it necessary to create a function, why just not put it in a variable?
```tsx
const selectedItemIndex = _value ? filteredData.findIndex((el) => el.value === _value) : -1
```

Reply:
hmm good point - tbh no idea why i did not do this as variable